compose/snippets/src/main/java/com/example/compose/snippets/glance/GlanceSnippets.kt

```diff
@@ -868,6 +870,34 @@ object GlanceTheming {
     }
 }
 
+object GlanceInnerPadding {
+
+    /**
+     * Applies corner radius for views that are visually positioned [widgetPadding]dp inside of the
+     * widget background.
+     */
+    @Composable
+    fun GlanceModifier.appWidgetInnerCornerRadius(): GlanceModifier {
+        // [START android_compose_glance_innercornerradius]
+        if (Build.VERSION.SDK_INT < 31) {
+            return this
+        }
+
+        // Padding around the widget
+        val widgetPadding = 12.dp
```

Comment:
Should this be passed in to the modifier?

Reply:
Sounds good!